api/proto/config/vm.proto

```diff
@@ -41,4 +41,9 @@ message VmConfig {
   uint32 vncDisplay = 17;
   string vncPasswd = 18;
   bool disableLogs = 19;
+  // Set if the VM should run in a mode with pinned CPUs. In this case all the
+  // VCPUs threads created by QEMU run on a dedicated physical CPU. All the
+  // other QEMU threads will be limited to run on the physical CPUs defined by
+  // CPU mask.
```

Comment:
Curious, do you plan to define `CPU mask` option somewhere separately, or it will be calculated somehow in code? Do we need to mention an information about non-vcpu threads here?

Reply:
"CPU mask." --> Is it better to indicate that this is referring to "cpus" string ?

Reply:
> "CPU mask." --> Is it better to indicate that this is referring to "cpus" string ? Also indicate the expected behavior when "cpus" is an empty string.

Done